.travis.yml

```diff
@@ -3,12 +3,13 @@ language: ruby
 sudo: false
 
 rvm:
+  - 2.3.1
   - 2.3.0
-  - 2.2
-  - 2.1
-  - 2.0.0
+  - 2.2.5
```

Comment:
I think we don't need to build all those, how about just the latest 2.2 (does just 2.2 work)?